frontend/helpers/constants.js

```diff
@@ -70,8 +70,8 @@ export const API_ENDPOINTS = {
   }
 };
 
-export const LC_PUBLISHED_PAGE = '/en/facilitator/study_group/published';
-export const LC_SAVED_DRAFT_PAGE = '/en/facilitator/study_group/saved';
+// export const LC_PUBLISHED_PAGE = '/en/facilitator/study_group/published';
+// export const LC_SAVED_DRAFT_PAGE = '/en/facilitator/study_group/saved';
```

Comment:
@dirkcuys do you know what happened to these pages? They're gone now?

Reply:
I don't think we ever used them. At some stage we created a success page, but then we decided it didn't make much sense since we're already sending them the info via email.